pkg/quota/image/imagestream_evaluator.go

```diff
@@ -38,15 +42,52 @@ func NewImageStreamEvaluator(osClient osclient.Interface) kquota.Evaluator {
 	return quotautil.NewSharedContextEvaluator(
 		"ImageStream evaluator",
 		kapi.Kind("ImageStream"),
+		nil,
+		computeResources,
+		matchesScopeFunc,
+		getFuncByNamespace,
+		listFuncByNamespace,
+		imageStreamConstraintsFunc,
+		makeImageStreamUsageComputerFactory(osClient),
+		imageapi.ResourceImageStreamSize,
+		imageapi.ResourceImageSize)
+}
+
+// NewImageStreamAdmissionEvaluator computes resource usage of ImageStreams in the context of admission
+// plugin.
+func NewImageStreamAdmissionEvaluator(osClient osclient.Interface) kquota.Evaluator {
+	computeResources := []kapi.ResourceName{
```

Comment:
This method could just call `NewImageStreamEvaluator` and set only the fields it needs to change to reduce some of the duplicate code